src/robot_gnss/test.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+# Тест получения данных с GNSS модуля
+# Скрипт копит значения и выдает среднее арифметическое
+
+import rospy
+from numpy import isnan, mean
+from sensor_msgs.msg import NavSatFix
+
+list_size = 300
+count = 0
+list_latitude = [0] * list_size
+list_longitude = [0] * list_size
+
+
+def nav_cb(nsf):
+    global count
+    print(f'[{count}]')
+    if count < list_size and not isnan(nsf.latitude):
+        print(f'{nsf.latitude}    {nsf.longitude}')
```

Comment:
Is there really that much space between the curly braces?

Reply:
Yes, so that latitude and longitude are well readable, I separate them with four spaces